general/quick-start.md

```diff
@@ -7,7 +7,10 @@ title: Quick Start
 
 1. Install Jellyfin on [your system](xref:admin-installing) with the installation method for your platform.
 
-1. [OPTIONAL from 10.7.0 onwards] [Edit the Jellyfin web configuration](xref:web-config) and adjust the options to fit your desired privacy level; our defaults sacrifice some absolute self-hosting for well-requested features, and if this is concerning please review the documentation and edit accordingly.
+1. Edit the [web configuration](xref:web-config) and adjust the options to fit your desired privacy level.
```

Comment:
if i disable the YouTube plugin in the web config  can't i bypass this with a standalone web?

Reply:
Yes, but you can always bypass client features with other clients.